api/cluster.capnp

```diff
@@ -6,6 +6,29 @@ $Go.package("cluster");
 $Go.import("github.com/wetware/ww/internal/api/cluster");
 
 
+interface Anchor {
+    ls @0 (path :List(Text), handler :Handler) -> ();
+    walk @1 (path :List(Text)) -> (anchor :Anchor);
+
+    interface Handler {
+        handle @0 (anchors :List(AnchorElement)) -> ();
+
+        struct AnchorElement{
```

Comment:
I think we should call this type `Child`.  The generated Go type will be called `Anchor_Child`, which seems adequately descriptive.